cli/tests/test_gateway.py

```
@@ -185,6 +191,93 @@ class OrchestratorClientWireFormatTests(unittest.TestCase):
+            def log_message(self, format: str, *args: object) -> None:
+                pass
+
+        origin_server = ThreadingHTTPServer(("127.0.0.1", 0), OriginHandler)
+        origin_thread = threading.Thread(target=origin_server.serve_forever, daemon=True)
+        origin_thread.start()
+
+        try:
+            client = gateway.OrchestratorClient(
+                host="127.0.0.1",
+                port=origin_server.server_port,
+                token="gateway-secret",
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to argument: "token"

(S106)